docs/settingup.md

```diff
@@ -2039,6 +2039,25 @@ Upload a data file to the Data manager.
 }
 ```
 
+</details><details>
+<summary>disconnectelastic</summary>
+
+## DisconnectElastic action
+
+Disconnect from an elastic environment. This action will disconnect open websockets towards sense and events. The action is not needed for most scenarios, however if a scenario mixes "elastic" environments with Sense for Windows or uses custom actions towards other type of environments it should be used directly after the last action towards the elastic environment.
```

Comment:
QSEoK deployment
QSEoW
or write it out Qlik Sense Enterprise on Windows/Kubernetes

other type**s** of environments**,** it